tests/system_tests_autolinks.py

```diff
@@ -35,6 +36,124 @@
 CONNECTION_PROPERTIES = {u'connection': u'properties', u'int_property': 6451}
 
 
```

Comment:
Please also provide a test which receives attaches for pre-configured auto-links.  Have the test reject the auto-links and see that they retry.  Also, have the test accept the auto-link and then detach it to see the retry.